inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/LoadNode.java

```diff
@@ -72,7 +73,8 @@
         @JsonSubTypes.Type(value = OracleLoadNode.class, name = "oracleLoad"),
         @JsonSubTypes.Type(value = GreenplumLoadNode.class, name = "greenplumLoad"),
         @JsonSubTypes.Type(value = DLCIcebergLoadNode.class, name = "dlcIcebergLoad"),
-        @JsonSubTypes.Type(value = DorisLoadNode.class, name = "dorisLoad")
+        @JsonSubTypes.Type(value = DorisLoadNode.class, name = "dorisLoad"),
+        @JsonSubTypes.Type(value = InfluxDBLoadNode.class, name = "influxdb")
```

Comment:
influxdb -> influxdbLoad

Reply:
1. Name changes done.
2.` the package of jdbc/src/main... should not be under sort-connectors.`, do you mean `the package of influxdb-jdbc/src/main`. And please explain why this is wrong.
3. this CDC sues JDBC table API: `options.put("connector", "jdbc-inlong");`
4. @jun0315  is already testing the referenced influx-jdbc-driver connector

Reply:
2.If it is based on the jdbc connector extension, you can put the influxdb-jdbc code in the jdbc connector, there is no need to start a single module
3.If it already have the unit test, you can put the corresponding link.

Reply:
> 2.If it is based on the jdbc connector extension, you can put the influxdb-jdbc code in the jdbc connector, there is no need to start a single module

I have a problem with this, this is a jdbc-driver. rather than a jdbc table API. The JDBC-connector module calls the influx-jdbc-driver module. When the influx jdbc driver is officially supported, we should delete this module and replace it with a new jdbc-driver. Writing them together not only makes no sense but also increases code coupling.

Reply:
The Sort Connectors module contains all connectors or modules common to connectors. There is currently no official driver implementation for influx db. It is reasonable to put the corresponding driver implementation in the jdbc connector. When the official implementation is available later, remove it.